scripts/upgrade

```diff
@@ -162,8 +162,15 @@ ynh_add_systemd_config
 # SECURE FILES AND DIRECTORIES
 #=================================================
 
-# Set right permissions for curl installation
-chown -R $app: $final_path
+# Reset permissions
+chown -R root:root "$final_path"
```

Comment:
I do not see this in the install script. Did you miss to put it ?

Reply:
this isn't needed anymore, we now use a dedicated user for each app:

```
$ grep -iR "chown" ./*
./scripts/restore:chown -R "$app": "$final_path"
./scripts/upgrade:chown -R "$app": "$final_path"
./scripts/install:chown -R "$app": "$final_path"
```